### PR TITLE
Add restart to webserver & api ctl namespace

### PR DESCRIPTION
**Pull request by fruttasecca**
## Description

Enables GUI `RESTART`.

### CODE REVIEW

File: services/orchest-api/app/app/apis/namespace_ctl.py [hunk 1]
```diff
@@ -40,12 +40,25 @@ def post(self):
...
 
 
+@api.route("/restart")
+class Restart(Resource):
+    @api.doc("orchest_api_restart")
+    def post(self):
+        update_pod_manifest = _get_restart_pod_manifest()
+        k8s_core_api.create_namespaced_pod(
+            _config.ORCHEST_NAMESPACE, update_pod_manifest
+        )
```
Comment (yannickperrenet) [inline services/orchest-api/app/app/apis/namespace_ctl.py:50]: `update_pod_manifest` --> `restart_pod_manifest`

File: services/orchest-webserver/app/app/views/views.py [hunk 2]
```diff
@@ -183,6 +183,13 @@ def server_config():
             }
         )
 
+    @app.route("/async/restart", methods=["POST"])
+    def restart():
+        resp = requests.post(
+            f'http://{current_app.config["ORCHEST_API_ADDRESS"]}/api/ctl' "/restart"
```
Comment (yannickperrenet) [inline services/orchest-webserver/app/app/views/views.py:189]: Probably due to black but can be changed to:

`f'http://{current_app.config["ORCHEST_API_ADDRESS"]}/api/ctl/restart'`

File: services/orchest-api/app/app/apis/namespace_ctl.py (hunk 1 above)
Comment (fruttasecca) [reply services/orchest-api/app/app/apis/namespace_ctl.py:50]: https://github.com/orchest/orchest/pull/772

File: services/orchest-webserver/app/app/views/views.py (hunk 2 above)
Comment (fruttasecca) [reply services/orchest-webserver/app/app/views/views.py:189]: https://github.com/orchest/orchest/pull/772

**Pull request merged**